File: java_projects/Package/par_contador.java
package Package;

import java.util.*;

public class par_contador {
    public static void main(String[] args) {
        Scanner leer = new Scanner(System.in);
        int contador = 0;
        int numero = 1;

        while (true){
            String numeroString = leer.nextLine();
            numero = Integer.parseInt(numeroString);
            int resto = numero % 2;
            if (numero == 0) {
            	break;
            }
            if (resto == 0){
                System.out.println("Par");
                contador = contador +1;
            } else if (resto != 0){
                System.out.println("Impar");
            }
        }
        System.out.println("La cantidad de números pares es " + contador);
    }
}

File: par_contador.py
numero = 1
contador = 0
while True:
    numero = int(input())
    resto = numero % 2
    if numero == 0:
        break
    if resto == 0:
        print("Par")
        contador = contador +1
    elif resto != 0:
        print("Impar")
print("La cantidad de números pares es " + str(contador))




